templates/verify.txt

```diff
@@ -1,7 +1,13 @@
-{{t "Welcome!"}}
+{{t "Ready, set sync"}}
 
-{{{t "Please activate your account by confirming this email address." }}}
-{{t "Activate now:"}} {{{link}}}
+{{{t "Confirm you’ve received this email and we’ll help you install and sync Firefox on all your devices starting with:" }}}
+
+{{ device }}
+{{#if location}}{{ location }}{{/if}}
+{{#if ip}}{{t "IP address: %(ip)s" }}{{/if}}
+{{#if timestamp}}{{ timestamp }}{{/if}}
```

Comment:
let's not put the timestamp in the verify email because @ryanfeeley does not have that one in the template. Other 2 emails should have it though